openwisp_monitoring/monitoring/configuration.py

```diff
@@ -174,6 +174,56 @@
             }
         },
     },
+    'signal_strength': {
+        'label': _('Signal Strength'),
+        'name': '{name}',
+        'key': '{key}',
+        'field_name': 'signal_strength',
+        'related_fields': ['signal_power'],
+        'charts': {
+            'signal_strength': {
+                'type': 'scatter',
+                'title': _('Signal Strength: {metric.key}'),
+                'label': _('Signal Strength'),
+                'colors': DEFAULT_COLORS,
```

Comment:
Wanted some help here. Without this line, the chart shows the same color for both lines. I observed that other charts (like RTT) show different colors for each line without adding this explicitly. What could possibly be wrong?
<details open>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/10897423/116823512-96eb8080-aba2-11eb-8e3d-0600935e3e24.png" />
</details>